Makefile.module

```diff
@@ -4,7 +4,11 @@ viam-mlmodelservice-triton.sh: bin/viam-mlmodelservice-triton.sh.envsubst
 	cat $< | TAG=$(TAG) DOLLAR=$$ envsubst > $@
 	chmod +x $@
 
-module.tar.gz: viam-mlmodelservice-triton.sh
+first_run.sh: bin/first_run.sh.envsubst
```

Comment:
There's probably a way to make a `*.sh` rule that unifies lines 3-9 into a single rule. Until there are 3 or 4 of these things, I'm inclined not to bother with that generalization, though.

Reply:
...okay, I did it already.